cmd/kubeadm/app/cmd/init.go

```diff
@@ -138,14 +142,15 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 		"The discovery method kubeadm will use for connecting nodes to the master",
 	)
 
-	return cmd
-}
+	cmd.PersistentFlags().StringVar(
+		&deploymentType, "deployment", deploymentType,
```

Comment:
@luxas is it enough to just add to kubeadm API or do I need to add it to `k8s.io/client-go` and `k8s.io/kubernetes/staging` as well?